collector/fixtures/proc/mdstat

```diff
@@ -11,18 +11,22 @@ md0 : active raid1 sdi1[0] sdj1[1]
 md4 : inactive raid1 sda3[0] sdb3[1]
       4883648 blocks [2/2] [UU]
 
-md6 : active raid1 sdb2[2] sda2[0]
+md5 : active raid1 sdb2[2] sda2[0]
```

Comment:
Please leave these unchanged, so that the diff is easier to read. Maybe name the new one `md5`.

I would prefer do this cleanup as a separate noop PR for the end-to-end fixture changes.

Reply:
Ok, I'll split my commit up.